packages/dds/shared-object-base/src/test/attachingBindingAndConnecting.spec.ts

```diff
@@ -27,7 +27,7 @@ import { createChildLogger } from "@fluidframework/telemetry-utils/internal";
 import type { IFluidSerializer } from "../serializer.js";
 import { SharedObject } from "../sharedObject.js";
 
-/* eslint-disable-next-line @typescript-eslint/ban-types --
+/* eslint-disable-next-line @typescript-eslint/no-unsafe-function-type --
```

Comment:
How come we don't need the `ban-types` disable anymore? I thought it would go away with the plugin upgrade.